hasura-api/metadata-json/unified.json

```diff
@@ -2335,7 +2335,8 @@
                     "sender",
                     "sequence_number",
                     "timestamp",
-                    "version"
+                    "version",
+                    "replay_protection_nonce"
```

Comment:
we have a process to update this. https://app.clickup.com/9011498509/v/dc/8cj13gd-14211/8cj13gd-8791

our oncall might be able to help here though.